### PR TITLE
Add backup paths for system info

### PR DESCRIPTION
**Pull request by Camerooooon**
WIP Don't merge

### CODE REVIEW

File: src/power.rs [hunk 1]
```diff
@@ -81,3 +97,14 @@ pub fn read_power_source() -> Result<bool, Error> {
...
+
+pub fn get_best_path(paths: Vec::<&str>) -> Result<&str, Error> {
+
+    for path in paths.iter() {
+        if Path::new(path).exists() {
+            return Ok(path);
+        }
+    }
+
+    return Err(Error::Unknown);
```
Comment (Shuzhengz) [inline src/power.rs:109]: Maybe add a thing to let the user know that their system is unsupported and tell them to submit an issue in the repo

Comment (Camerooooon) [reply src/power.rs:109]: Great idea!

Comment (JakeRoggenbuck) [reply src/power.rs:109]: Should we keep the little script in the github gist or should we move it this repo? I'm good with either

**Comment by Camerooooon** [comment]
@JakeRoggenbuck @Shuzhengz Please test this on your computers and let me know if it works.

**Comment by Camerooooon** [comment]
Works on my Razer Blade and XPS. Also Jake can you try to get Milo to test this?

**Review by Shuzhengz — APPROVED** [review]
Works on my X1 Extreme

**Comment by Shuzhengz** [comment]
2 of my 12 cores report 0 tempature, but i think that's a thing with my CPU so were good

**Comment by Camerooooon** [comment]
> 2 of my 12 cores report 0C tempature, but i think that's a thing with my CPU so were good

Can you try checking the actual file path for the temperature? I never actually added backup strings for CPU temperature zones I don't think

**Comment by Shuzhengz** [comment]
It's the same as the current one

**Comment by JakeRoggenbuck** [comment]
Works! My dell latitude, found battery and changed to performance when plugged in, so it found my ac

**Comment by JakeRoggenbuck** [comment]
How to test
```
git fetch origin pull/115/head
git checkout FETCH_HEAD
cargo build
cargo run monit
# check for battery and gov mode change on plugin
```

**Review by JakeRoggenbuck — APPROVED** [review]
Thank you so much @Camerooooon!!

**Comment by Camerooooon** [comment]
Awesome! This is ready to merge.

**Pull request merged**